webcompat/views.py

```diff
@@ -306,3 +321,13 @@ def contributors():
 def cssfixme():
     '''Route for CSS Fix me tool'''
     return render_template('cssfixme.html')
+
+
+@app.route('/csp-report', methods=['POST'])
+def log_report():
+    '''Route to record CSP header violations.'''
+    if 'application/csp-report' not in request.headers.get('content-type'):
+        return ('Wrong Content-Type.', 400)
+    with open(app.config['CSP_REPORTS_LOG'], 'a') as r:
+        r.write(request.data + '\n')
```

Comment:
So my suggestion was a bad idea, given the context of flask which is exploding the normal logging python module. The [logging module](https://docs.python.org/2/library/logging.html) is ["simple"](https://pymotw.com/3/logging/index.html), but flask makes it more difficult. Cf. https://github.com/pallets/flask/issues/2023

so let's go with your initial suggestion.